diff --git a/examples/range/range.ts b/examples/range/range.ts
--- a/examples/range/range.ts
+++ b/examples/range/range.ts
@@ -106,6 +106,6 @@ async function setupWorld() {
   gameContext.renderer = new FuseRenderer({
     canvas,
     antialias: true,
-    renderMode: 'webgpu',
+    renderMode: 'webgl',
     qualityLevel: gameContext.setupData?.qualityLevel ?? 2
   });
@@ -132 +132 @@ async function createGroundPlane() {
-  grassTexture.anisotropy = gameContext.renderer?.getMaxAnisotropy() || 1;
+  grassTexture.anisotropy = Math.min(gameContext.renderer?.getMaxAnisotropy() || 1, 4);
@@ -139 +139 @@ async function createGroundPlane() {
-  grassNormalMap.anisotropy = gameContext.renderer?.getMaxAnisotropy() || 1;
+  grassNormalMap.anisotropy = Math.min(gameContext.renderer?.getMaxAnisotropy() || 1, 4);
